public/js/helpers/humanize_helper.js

```diff
@@ -138,25 +138,31 @@ var humanize = {
       }
       return result
     }
+    var pad = function (x) {
+      if (x < 10) {
+        return `0${x}`
+      }
+      return x
+    }
     interval = Math.floor(seconds / 3600)
     if (interval >= 1) {
       let extra = Math.floor((seconds - interval * 3600) / 60)
       let result = interval + 'h'
       if (extra > 0) {
-        result = result + ' ' + extra + 'm'
+        result = result + ' ' + pad(extra) + 'm'
       }
       return result
     }
     interval = Math.floor(seconds / 60)
     if (interval >= 1) {
       let extra = seconds - interval * 60
-      let result = interval + 'm'
+      let result = pad(interval) + 'm'
```

Comment:
I don't know that the minutes ever needs padding.  Also zeros don't look good here.  And if there are values >100m then the padding wouldn't be adequate anyway, no?

Reply:
There won't be numbers greater than 59m, that will round to 1h.

Reply:
For some reason I thought we had this with minutes as the largest unit.  Thanks for correcting me.